app/mailers/settings_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class SettingsMailer < ApplicationMailer
+  
+  def update_email(user)
```

Comment:
This could have just as easily been placed in the UserMailer instead of creating a SettingsMailer.  Works just as well this way though!